x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v7_12_0.ts

```diff
@@ -6,15 +6,13 @@
  */
 
 import type { SavedObjectMigrationFn, SavedObjectUnsanitizedDoc } from '@kbn/core/server';
-import { cloneDeep } from 'lodash';
 
 import type { PackagePolicy } from '../../../../common';
 
 export const migratePackagePolicyToV7120: SavedObjectMigrationFn<PackagePolicy, PackagePolicy> = (
   packagePolicyDoc
 ) => {
-  const updatedPackagePolicyDoc: SavedObjectUnsanitizedDoc<PackagePolicy> =
-    cloneDeep(packagePolicyDoc);
+  const updatedPackagePolicyDoc: SavedObjectUnsanitizedDoc<PackagePolicy> = packagePolicyDoc;
```

Comment:
@parkiino I think it's OK to remove the usage of `cloneDeep` in the latest migration, but I think we should leave the older ones alone.

Reply:
++, let's not change existing migrations, especially given they were already released. Was only asking about the this PR introduced.

Reply:
ok i'll revert those!